src/Components/RideProgressCard/RideProgressCard.story.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { RideProgressCard } from "../RideProgressCard";
+
+const stories = storiesOf("Ride Progress Card", module);
+
+const origin = {
+  time: "08:15",
+  day: "10 November",
+  city: "Paris, France",
+  address: "84, Ave des Champs-Elisey",
+};
+const destination = {
+  time: "16:45",
+  day: "02 December",
+  city: "Nantes, France",
+  address: "Paran 3, 615 A",
+};
+const progress = 10;
+
+stories.add("Ride Progress Card", () => (
```

Comment:
Storybooks are need to illustrate all the difficult use-cases of the compoent, e.g. too long text